packaging/darwin/amazon-cloudwatch-agent-ctl

```diff
@@ -252,23 +255,87 @@ cwa_config() {
      fi
 }
 
+getopts_long() {
```

Comment:
If this is what is required in order to support using `-strict` as a flag, I don't think it's worthwhile. See previous comment about looking at alternative single char flags to use other than `-v`

Reply:
That's not only the reason why I support using long arguments though. Another reason for this is when using `--help`, it would duplicate the `UsageString` with the error invalid option. Moreover, using long arguments also indicates their meaning. So in the future, we would need to support long arguments as well.

Reply:
I understand why we would want to include it. I guess the question is, do we want to include that in this specific PR or address it separately? That's what I want some outside opinions on.